admin-dev/themes/default/scss/controllers/_cartrules.scss

```diff
@@ -0,0 +1,11 @@
+.admincartrules {
+  #toolbar-footer {
+    overflow: visible;
+
+    .hidden-xs {
```

Comment:
No I need to target a specific id in desktop mode

Reply:
I understand, so you should add a class `.desktop-buttons` and `.mobile-buttons` and use `:not(:last-child)` to apply the margin to all buttons except the right one. 👍

This is exactly the kind of hackjobs that we want to avoid and I was removing in my PRs.

Reply:
Approving, please don't! Add a specific class for the desktop and mobile button :)

Reply:
Fixed.